notebooks/WFC3/point_spread_function/psf_utilities.py

```diff
@@ -0,0 +1,402 @@
+import os
+import shutil
+import urllib
+import numpy
+from numpy import percentile
+import matplotlib
+import matplotlib.pyplot
+from matplotlib.colors import LogNorm
+from astropy.io import fits
+from astropy.visualization import simple_norm
+from astropy.stats import sigma_clipped_stats
+from photutils.detection import find_peaks
+from photutils.centroids import centroid_com
+from photutils.aperture import CircularAperture
+from scipy.ndimage import shift
+
+
+def save_figure(figure, filename, show_figure):
+
+    '''
+    Save the current figure and display or close the file.
+    '''
+
+    figure.tight_layout()
+    matplotlib.pyplot.savefig(filename)
+    if (show_figure is True):
+        matplotlib.pyplot.show()
+    if (show_figure is False):
+        matplotlib.pyplot.close()
+
+    return print('\nFigure saved as: '+filename)
```

Comment:
Use f-string, also no need to return since it's not a variable that needs to be used later on
`print(f'\nFigure saved as: {filename}')`

Reply:
In that case, would it be best practice to return nothing, or some other form of status message?

Reply:
If you want `save_figure` to return a variable that will be used somewhere else, then return that. If not, it's best practice to simply print your statement and return nothing.

Reply:
Done.